### PR TITLE
Adds BlueBubbles entry to ADOPTERS.md

### PR DESCRIPTION
**Pull request by zlshames**

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -45,3 +45,4 @@ Here's the list of projects which are proudly adopting and using OpenZiti
...
 | Run Free                          | https://runfreeproject.com/                             | <img src="https://shop.runfreeproject.com/assets/icons/apple-icon-144x144.png" width="100px"> | The Run Free Project provides a premium ecommerce experience for Run Speciaty stores. After a variety of intrusion attempts on the database and web services, Run Free implemented a Ziti only access policy for interconnecting store data and services.  |
+| BlueBubbles                          | https://bluebubbles.app/                             | <img src="https://raw.githubusercontent.com/BlueBubblesApp/bluebubbles-server/master/icons/regular/icon-256.png" width="100px"> | BlueBubbles is an open-source app ecosystem dedicated to bringing iMessage to Android, Windows, and Linux. Shock your iPhone friends by iMessaging them from nearly any platform!  |
```
Comment (dovholuknf) [inline ADOPTERS.md:48]: The project appears to be open source primarily. Would you like to move it up the list to the "open source adopters" section at the top or would you prefer to keep it here? We have a section at the top promoting other open source projects, it's easy to miss :)

**Review by dovholuknf — APPROVED** [review]
thanks!

**Comment by zlshames** [comment]
Alright, should be fixed now. Thanks for the heads-up!

**Comment by dovholuknf** [comment]
unfortunately two of your commits were not signed for some reason (the first commit was). the project does require signed commits, enforced by github. Would you mind updating the PR so the commits are all signed?

**Comment by zlshames** [comment]
> unfortunately two of your commits were not signed for some reason (the first commit was). the project does require signed commits, enforced by github. Would you mind updating the PR so the commits are all signed?

I did it through GitHub's Codespaces... 🤦 idk why it wouldnt be signed... gimme a sec

**Comment by zlshames** [comment]
I'm going to close this and start a new PR. Had to delete and re-create my codespace and it screwed everything up...

**Pull request closed without merging**